mmv1/products/artifactregistry/Repository.yaml

```diff
@@ -527,6 +531,14 @@ properties:
                   remote repository. Must be in the format of
                   `projects/{project}/secrets/{secret}/versions/{version}`.
                 immutable: true
+      - !ruby/object:Api::Type::Boolean
+        name: 'disableUpstreamValidation'
```

Comment:
Is this field updatable? If so, would you mind adding it to a existing update test by testing the field setting `true` -> `false`. https://googlecloudplatform.github.io/magic-modules/develop/test/test/#add-an-update-test. Thanks!

Reply:
This field is a little weird because it is INPUT_ONLY. AR doesn't store its value or use it after the initial request. If this value was changed from true -> false, AR will understand the request and behave the same way as if the value was originally false.

Technically we do allow a user to disable upstream validation on an UpdateRepository request to change remote auth, but that is not typical behavior (if a user is changing a bad auth configuration, normally they wouldn't want to disable the check verifying the auth works).

I'm not sure it makes much sense to add an update test for this, but I'd also prefer not to make it immutable in case for some reason a user originally set it to false (default value) and wanted to change it to true. But maybe it makes more sense to make this immutable? WDYT?